common/src/com/thoughtworks/go/config/materials/Materials.java

```diff
@@ -140,9 +134,9 @@ public void cleanUp(File baseFolder, ConsoleOutputStreamConsumer consumer) {
         return allowed;
     }
 
-    boolean hasOneMaterialUseBaseFolder() {
+    public boolean hasOneMaterialUseBaseFolder() {
         for (Material material : this) {
```

Comment:
Sure, can do that. I just wanted to have extensive unit test around it. But I agree with you.